.github/ISSUE_TEMPLATE/offboarding.md

```diff
@@ -72,33 +88,47 @@ ticket has been completed. At that time, put the ticket back to triage.
+- [ ] Assigned themselves the `GitLab own` role on the `prod` instance of GitLab <sub>or the offboarded employee/contractor did not perform the `system administrator` role</sub>
+- [ ] Assigned themselves the `GitLab own` role on the `anvildev` instance of GitLab <sub>or the offboarded employee/contractor did not perform the `system administrator` role</sub>
+- [ ] Assigned themselves the `GitLab own` role on the `anvilprod` instance of GitLab <sub>or the offboarded employee/contractor did not perform the `system administrator` role</sub>
+- [ ] Removed the offboarded employee/contractor from the list of members on the Group members page in the `dev` instance of GitLab
+- [ ] Removed the offboarded employee/contractor from the list of members on the Group members page in the `prod` instance of GitLab
+- [ ] Removed the offboarded employee/contractor from the list of members on the Group members page in the `anvildev` instance of GitLab
+- [ ] Removed the offboarded employee/contractor from the list of members on the Group members page in the `anvilprod` instance of GitLab
+- [ ] Set the offboarded employee/contractor's account status to `Blocked` on the Users page in the Admin Area of the `dev` instance of GitLab
+- [ ] Set the offboarded employee/contractor's account status to `Blocked` on the Users page in the Admin Area of the `prod` instance of GitLab
+- [ ] Set the offboarded employee/contractor's account status to `Blocked` on the Users page in the Admin Area of the `anvildev` instance of GitLab
+- [ ] Set the offboarded employee/contractor's account status to `Blocked` on the Users page in the Admin Area of the `anvilprod` instance of GitLab
 - [ ] Removed all role assignments from the offboarded employee/contractor in the `dev` instance of GitLab
 - [ ] Removed all role assignments from the offboarded employee/contractor in the `prod` instance of GitLab
 - [ ] Removed all role assignments from the offboarded employee/contractor in the `anvildev` instance of GitLab
 - [ ] Removed all role assignments from the offboarded employee/contractor in the `anvilprod` instance of GitLab
-- [ ] Blocked the GitLab account owned by the offboarded employee/contractor on every GitLab instance in the system
-- [ ] Removed the Google Service accounts owned by the offboarded employee/contractor in the `platform-hca-dev` Google Cloud project
-- [ ] Removed the Google Service accounts owned by the offboarded employee/contractor in the `platform-hca-prod` Google Cloud project
-- [ ] Removed the Google Service accounts owned by the offboarded employee/contractor in the `platform-anvil-dev` Google Cloud project
-- [ ] Removed the Google Service accounts owned by the offboarded employee/contractor in the `platform-anvil-prod` Google Cloud project
-- [ ] Performed an account review and disabled any accounts owned by the offboarded employee/contractor
```

Comment:
How is this item in the document being addressed, it seems like it went away from the checklist but is part of the security architecture document.

Reply:
The PR checklist is indented replace the need for an account review.
The removal/disabling of the various account types are covered in the sections listed below:

- UCSC account: Covered under `ITS actions`
- Google workspace account: Covered under `System administrator actions`
- Google mail account: Covered under `ITS actions` with the disabling the USCS account
- Google service account: Covered under `System administrator actions`
- Terra user account: Covered under `System administrator actions`
- GitHub user account: Covered under `System administrator actions` with the removal of the user from all GitHub organizations
- GitHub machine account: Coverage added to `Offboarded employee/contractor actions` and `System administrator actions`
- GitLab user account: Covered under `System administrator actions`
- GitLab root account: Covered under `System administrator actions` with the creation of new GitLab EC2 instances

Reply:
Ah, makes sense. Thank you for the detailed list. ☺️